server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -557,6 +713,10 @@ ClusterNode getAnyNodeExcept(ClusterNode... clusterNodes) {
             return randomFrom(acceptableNodes);
         }
 
+        public long getCurrentTimeMillis() {
```

Comment:
Hmm, this is so we can call `runUntil(getCurrentTimeMillis() + ...)` but that's always how we call `runUntil`. Maybe move the addition inside, call it `runFor(...)`, and adjust the logging suitably (cos searching for absolute times is a useful debugging tool).

Reply:
good idea